examples/terraform-aws-rds-example/main.tf

```diff
@@ -0,0 +1,78 @@
+data "aws_vpc" "default" {
+  default = true
+}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_subnet" "example_1" {
+  vpc_id     = "${data.aws_vpc.default.id}"
+  availability_zone = "${data.aws_availability_zones.available.names[0]}"
+}
+
+data "aws_subnet" "example_2" {
+  vpc_id     = "${data.aws_vpc.default.id}"
+  availability_zone = "${data.aws_availability_zones.available.names[1]}"
+}
```

Comment:
Instead of these three data sources, you could use _just_ the `aws_subnet_ids` data source. In the `aws_db_subnet_group`, you could then use `data.aws_subnet_ids.default.ids[0]` and `data.aws_subnet_ids.default.ids[1]`.